### PR TITLE
[DO NOT MERGE] POC Single Storage - upgrade script

### PR DESCRIPTION
**Pull request by jjanvier**
# DO NOT MERGE THIS PR

This branch offers the minimum code to be able to store:
* product values as a JSON field in MySQL
* the product identifier as a dedicated field

The script `upgrades/ValuesAsJson/store_values_as_json.php` allows to convert a regular PIM database to the one we want to POC.

Beware to clear the `prod` cache before launching this script:

```
php app/console cache:clear --env=prod &&\
php upgrades/ValuesAsJson/store_values_as_json.php
```

The branch `poc-single-storage` is based on this branch.

### CODE REVIEW

File: upgrades/ValuesAsJson/store_values_as_json.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+echo "Creating unique index on identifier...\n";
+createUniqueIndexOnIdentifier($db, $productTable);
+
+echo "Done :)\n";
+
+function prepareProductTable(Connection $db, $productTable)
+{
+    $db->exec("ALTER TABLE $productTable ADD identifier VARCHAR(255) NOT NULL");
+    $db->exec("ALTER TABLE $productTable ADD raw_values LONGTEXT NOT NULL COMMENT '(DC2Type:json_array)'");
```
Comment (jjanvier) [inline upgrades/ValuesAsJson/store_values_as_json.php]: BLOB

File: upgrades/ValuesAsJson/ProductValuesNormalizer.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Pim\Upgrade\ValuesAsJson;
```
Comment (jjanvier) [inline upgrades/ValuesAsJson/ProductValuesNormalizer.php]: drop it

File: upgrades/ValuesAsJson/store_values_as_json.php [hunk 3]
```diff
@@ -0,0 +1,73 @@
+<?php
+
+require_once __DIR__ . '/../../app/bootstrap.php.cache';
+require_once __DIR__ . '/../../app/AppKernel.php';
+require_once __DIR__ . '/../SchemaHelper.php';
+require_once __DIR__ . '/../UpgradeHelper.php';
+
+use Doctrine\DBAL\Driver\Connection;
+use Pim\Upgrade\SchemaHelper;
+use Pim\Upgrade\ValuesAsJson\ProductValuesNormalizer;
```
Comment (jjanvier) [inline upgrades/ValuesAsJson/store_values_as_json.php]: use the one that comes from the PIM

Comment (SamirBoulil) [inline upgrades/ValuesAsJson/store_values_as_json.php]: Used this one: Pim\Component\Catalog\Normalizer\Standard\Product\ProductValuesNormalizer

**Comment by nidup** [comment]
This PR can be closed?

**Pull request closed without merging**